### PR TITLE
[deckhouse-controller] Deckhouse release controller metrics

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Description
Deckhouse update metrics restored

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse-controller
type: fix
summary: Deckhouse update metrics restored.
impact_level: default
```

### CODE REVIEW

**Comment by libmonsoon-dev** [comment]
image

File: deckhouse-controller/pkg/controller/deckhouse-release/controller.go [hunk 1]
```diff
@@ -456,11 +474,17 @@ func (r *deckhouseReleaseReconciler) tagUpdate(ctx context.Context, pods []corev
...
+	r.metricStorage.CounterAdd("deckhouse_kube_image_digest_check_total", 1, map[string]string{})
+
 	repoDigest, err := regClient.Digest(tag)
 	if err != nil {
+		r.metricStorage.CounterAdd("deckhouse_registry_check_errors_total", 1, map[string]string{})
 		return fmt.Errorf("registry (%s) get digest failed: %s", repo, err)
 	}
 
+	r.metricStorage.GaugeSet("deckhouse_kube_image_digest_check_success", 1.0, map[string]string{})
+
```
Comment (miklezzzz) [inline deckhouse-controller/pkg/controller/deckhouse-release/controller.go:487]: not sure, but maybe it should've been of counter type? I mean this Gauge is going to be set to 1 once and for all..

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/gcp'...

**Comment by z9r5** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.27 10141539147 10141539147-1-con-1-27 /sys/deckhouse-oss/install:pr9147 user@34.89.159.20

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by z9r5** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 10142323506 10142323506-1-con-1-27 /sys/deckhouse-oss/install:pr9147 redos@89.169.143.156

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: deckhouse-controller/pkg/controller/deckhouse-release/controller.go (hunk 1 above)
Comment (libmonsoon-dev) [reply deckhouse-controller/pkg/controller/deckhouse-release/controller.go]: Fixed

**Comment by libmonsoon-dev** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.27 10158097384 10158097384-1-con-1-27 /sys/deckhouse-oss/install:pr9147 user@34.89.234.204

**Comment by libmonsoon-dev** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 10158098038 10158098038-1-con-1-27 /sys/deckhouse-oss/install:pr9147 redos@89.169.143.20

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've recognized your '/e2e/abort' command and started the workflow...

**Review by yalosev — APPROVED** [review]

**Pull request merged**